### PR TITLE
1 34 docs update

### PR DESCRIPTION
**Pull request by pliablepixels**
#2740

### CODE REVIEW

File: docs/userguide/definemonitor.rst [hunk 1]
```diff
@@ -46,6 +46,10 @@ Linked Monitors
     This field allows you to select other monitors on your system that act as triggers for this monitor. So if you have a camera covering one aspect of your property you can force all cameras to record while that camera detects motion or other events. You can either directly enter a comma separated list of monitor ids or click on ‘Select’ to choose a selection. Be very careful not to create circular dependencies with this feature however you will have infinitely persisting alarms which is almost certainly not what you want! To unlink monitors you can ctrl-click. 
 
 Maximum FPS 
+
+    .. warning::
+      Unless you know what you are doing, please leave this field empty, especially if you are configuring a network camera. More often than not, putting a value here adversely affects live viewing performance.
+
```
Comment (connortechnology) [inline docs/userguide/definemonitor.rst:52]: I would say affects recording performance as opposed to live viewing.  The actual recordings will be junk as ffmpeg is starved for cpu and can't keep up with receiving the stream.

File: docs/userguide/definemonitor.rst [hunk 2]
```diff
@@ -71,15 +80,15 @@ Source Tab
...
 
         * Check the documentation that came with your camera
-        * Look for your camera in the hardware compatibilty list in the wiki https://wiki.zoneminder.com/Hardware_Compatibility_List
+        * Look for your camera in the hardware compatibilty list in the `hardware compatibility wiki <https://wiki.zoneminder.com/Hardware_Compatibility_List>`__
         * Try ZoneMinder's new ONVIF probe feature
-        * Download and install the ONVIF Device Manager onto a Windows machine https://sourceforge.net/projects/onvifdm/
+        * Download and install the `ONVIF Device Manager <https://sourceforge.net/projects/onvifdm/>`__ onto a Windows machine 
         * Use Google to find third party sites, such as ispy, which document this information
 Source Colours 
     Specify the amount of colours in the captured image. 32 bit is the preferred choice here. Unlike with local cameras changing this has no controlling effect on the remote camera itself so ensure that your camera is actually capturing to this palette beforehand.
```
Comment (connortechnology) [inline docs/userguide/definemonitor.rst:94]: Note to Ic0n, rewrite this section to mention why we prefer 32 bit and make the local/network camera bit clearer.

File: docs/userguide/definemonitor.rst (hunk 1 above)
Comment (pliablepixels) [reply docs/userguide/definemonitor.rst:52]: resolved by https://github.com/pliablepixels/ZoneMinder/commit/723e973e8805b2cef1b012c578c5fd3f45ec7518

**Comment by pliablepixels** [comment]
I'm done with changes. @connortechnology to review.
Please also search for `.. todo::` in docs - they need your inputs.

File: docs/installationguide/easydocker.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+An Easy To Use Docker Image
+===========================
+If you are interested in trying out ZoneMinder quickly, user Dan Landon maintains an easy to use docker image for ZoneMinder. With a few simple configuration changes, it also provides complete Event Server and Machine Learning hook support. Please follow instructions in his repostory. He maintains two repositories:
+
+* If you want to run the latest stable release, please use his `zoneminder repository <https://github.com/dlandon/zoneminder>`__.
+* If you want to run the latest zoneminder master, please use his `zoneminder master repository <https://github.com/dlandon/zoneminder.master-docker>`__.
+
+In both cases, instructions are provided in the repo README files.
+
+If you are looking at making your own docker build of ZoneMinder instead, please refer to :doc:`packpack`.
```
Comment (connortechnology) [inline docs/installationguide/easydocker.rst]: packpack isn't really making a docker build of zoneminder... it is about using docker to build binary packages of ZoneMinder to install directly, without docker.  If people want to build their own docker build of zoneminder... they should start with dlandon's Dockerfile.

File: docs/userguide/configfiles.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
+Configuration Files
+--------------------
+This section describes configuration files that ZoneMinder users beyond the various Web UI options.
```
Comment (connortechnology) [inline docs/userguide/configfiles.rst]: uses instead of users

File: docs/api.rst [hunk 5]
```diff
@@ -97,26 +97,23 @@ Once you have the keys (a.k.a credentials (v1.0, v2.0) or token (v2.0)) you shou
...
 
   # or, if you specified -c cookies.txt in the original login request
 
   curl -b cookies.txt -XGET   https://yourserver/zm/api/monitors.json
 
 
 .. NOTE::
-	ZoneMinder's API layer allows API keys to be encoded either as a query parameter or as a data payload. If you don't pass keys, you could use cookies (not recommended as a general approach)
 
+	If you are using an ``HTTP GET`` request, the token/auth needs to be passed as a query parameter in the URL. If you are using an ``HTTP POST`` (like when you use the API to modify a monitor, for example), you can choose to pass the token as a data payload instead. The API layer discards data payloads for ``HTTP GET``. Finally, If you don't pass keys, you could also use cookies (not recommended as a general approach).
```
Comment (pliablepixels) [inline docs/api.rst:116]: https://github.com/ZoneMinder/zoneminder/issues/2744

**Pull request merged**